test/library/standard/LinkedLists/first-last-search.chpl

```diff
@@ -0,0 +1,17 @@
+use LinkedLists;
```

Comment:
The tests seems to look fine, though you can do some code-style improvements,  like spacing between operations between single, double... item lists, Adding comments in a separate line above. Though Chapel does not have an official style guide, you can do some improvements that make your code look clean. :-)